CosmosApiChangeFeedForMessageRetention/index.ts

```diff
@@ -39,14 +39,13 @@ const telemetryClient: TelemetryClient = initTelemetryClient(
 );
 
 const run: AzureFunction = async (
-  context: Context,
+  _: Context,
   documents: ReadonlyArray<RetrievedMessageStatus>

```

Comment:
We need to handle documents as `unknown`, and decode them before usage.
```suggestion
  documents: ReadonlyArray<unknown>

```